results/google__text-embedding-005/external/model_meta.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "google/text-embedding-005",
+    "revision": "external",
+    "release_date": "2024-11-18",
+    "languages": [
+        "eng-Latn"
+    ],
+    "n_parameters": null,
+    "memory_usage": null,
+    "max_tokens": 2048.0,
+    "embed_dim": 768,
+    "license": null,
+    "open_weights": false,
+    "public_training_data": null,
+    "public_training_code": null,
+    "framework": [
+        "API"
+    ],
+    "reference": null,
+    "similarity_fn_name": "cosine",
+    "use_instructions": true,
+    "training_datasets": null,
```

Comment:
If you know that the model is not trained on any of the dataset in mteb (including other benchmarks), we can add:

```suggestion
    "training_datasets": [],
```
In which case it will appear as a zero-shot otherwise it will state that the dataset is unknown in the v2 of the benchmark (this is the demo, i.e. not released yet):

![400787265-73ff3832-6208-4e65-b117-7f1929dd2228](https://github.com/user-attachments/assets/9e525654-7911-4f45-a6b5-088c2a17633e)

Reply:
The training datasets haven't been revealed, so I will keep it as "null" for now.